### PR TITLE
add recipe for neworder (from pypi)

### PR DESCRIPTION
**Pull request by virgesmith**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/neworder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  imports:
+    - neworder
+  commands:
+    - pip check
+    - pytest
+  requires:
+    - pip
+    - pytest
+    - numpy
+    - pandas
```
Comment (scopatz) [inline recipes/neworder/meta.yaml:40]: the only entry needed here is pytest

Comment (virgesmith) [reply recipes/neworder/meta.yaml:40]: ok thanks will remove and reinstate pytest with verbose flags, as I don't quite get why the logs say no tests are being run:

```
============================= test session starts ==============================
platform linux -- Python 3.6.11, pytest-6.0.2, py-1.9.0, pluggy-0.13.1
rootdir: $SRC_DIR
collected 0 items
```

Comment (virgesmith) [reply recipes/neworder/meta.yaml]: now saying 
`home/conda/staged-recipes/build_artifacts/neworder_1600287643571/test_tmp/run_test.sh: line 7: pip: command not found`
so going to add pip back to requres

Comment (scopatz) [reply recipes/neworder/meta.yaml]: hmm weird yeah

**Comment by scopatz** [comment]
Let us know when this is ready!

**Comment by virgesmith** [comment]
Ready. (it tested in a conda env here: https://travis-ci.org/github/virgesmith/neworder, so having to remove pytest here isnt a showstopper)

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**